app/utilities/schema.py

```diff
@@ -25,6 +27,41 @@
 }
 
 
+def get_schema_path_map_for_language(language_code):
+    schema_dirs = [
+        os.path.join(schema_dir, language_code) for schema_dir in SCHEMA_DIRS
+    ]
+
+    schema_files = []
+
+    for schema_dir in schema_dirs:
+        schema_files.extend(glob(f"{schema_dir}/*.json"))
+
+    return {
+        Path(schema_file).with_suffix("").name: schema_file
+        for schema_file in schema_files
+    }
+
+
+def get_schema_path_map():
+    language_map_codes = ["en", "cy", "ga", "eo"]
+
+    return {
+        language_code: get_schema_path_map_for_language(language_code)
+        for language_code in language_map_codes
+    }
+
```

Comment:
This can be  made more succinct by using the Path object.

```Python
from collections import ChainMap

def schema_path_maps_for_language(language_code):
    schema_dirs = ["schemas", "test_schemas"]
    
    for schema_dir in schema_dirs:
            yield {Path(p).with_suffix('').name: p for p in list(Path(schema_dir, language_code).glob('*.json'))}

def get_schema_path_map():
    language_map_codes = ["en", "cy", "ga", "eo"]

    return {
        language_code: ChainMap(*get_schema_path_map_for_language(language_code))
        for language_code in language_map_codes
    }
```

Reply:
This may be more succinct, but introduces quite a lot of complexity. I'm not sure it's worth the confusion.

Reply:
I find it easier to read.

Which lines are overly complex? It may be that there is a way to reduce the complexity and keep the more succinct approach.

Reply:
Just saw the latest change - I think that's a good approach and answers the above question :+1: